ndless-sdk/include/libndls.h

```diff
@@ -102,10 +102,23 @@ BOOL isKeyPressed(const t_key *key);
 void wait_key_pressed(void);
 void wait_no_key_pressed(void);
 /* config.c */
+struct cfg_entry {
+	char key[15];
```

Comment:
Fixed size?

Reply:
I thought this was OK because the only place that wrote to the cfg in code (`cfg_register_fileext`) also limited the key size to 15. However, this doesn't take into account manual editing of the config file

Reply:
Yes, for the fileext it might apply, but not for other keys. Part of this effort of refactoring this is to make the code more versatile, isn't it? :-)

Reply:
How would one make this a non-static size? A variable and a helper function?

Reply:
Most likely, yes. Alternatively make the fixed size bigger and provide it as a global constant.